qr_generator/lib/data/sqlite_persistence.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:sqflite/sqlite_api.dart';
+
+class SqlitePersistence {
+  static const databaseName = 'superformula.db';
+  static const seedTableName = 'seed';
+  Database db;
```

Comment:
**suggestion**: make this field `final` so that we can also have a `const` constructor!

```suggestion
  final Database db;
```

Reply:
Done!